### PR TITLE
revise imports after forte-wrapper rename

### LINKED ISSUE #16: Revise imports and dependencies after forte-wrapper rename

### PR DESCRIPTION
**Pull request by jzpang**
This PR fixed https://github.com/petuum/composing_information_system/issues/16.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,4 +5,5 @@ git+https://git@github.com/asyml/forte-wrappers.git#egg=forte.nltk&subdirectory=
 git+https://git@github.com/asyml/forte-wrappers.git#egg=forte.elastic&subdirectory=src/elastic
 git+https://git@github.com/asyml/forte-wrappers.git#egg=forte.allennlp&subdirectory=src/allennlp
 git+https://git@github.com/asyml/forte-wrappers.git#egg=forte.spacy&subdirectory=src/spacy
+git+https://git@github.com/asyml/forte-wrappers#egg=forte-wrappers[scispacy]
```
Comment (hunterhector) [inline requirements.txt]: We will probably not do it this way.

Could you help add `scispacy` as an extra requirement for spacy here: https://github.com/asyml/forte-wrappers/blob/main/src/spacy/setup.py

Comment (hunterhector) [reply requirements.txt]: This file should be deleted (I forgot): https://github.com/asyml/forte-wrappers/blob/main/setup.py, you could do it together with the suggestion above.

**Review by hunterhector — COMMENTED** [review]
is it possible to add simple tests to ensure the imports?

File: requirements.txt (hunk 1 above)
Comment (jzpang) [reply requirements.txt]: I tried to add scispacy in spacy setup.py and delete setup.py in main: https://github.com/asyml/forte-wrappers/tree/twitter_rename, but the pipeline failed at mypy check. I will try to find out a way to make it back to work.

Comment (hunterhector) [reply requirements.txt]: I think we just run mypy multiple times?  Like `mypy src/allennlp`, you can loop over the directories: https://github.com/asyml/forte-wrappers/blob/main/.github/workflows/main.yml#L156

**Comment by hunterhector** [comment]
fixes are the same as https://github.com/petuum/composing_information_system/pull/15, closing.

**Pull request closed without merging**